cli_reference/developer_cli_odo/odo-release-notes.adoc

```diff
@@ -7,78 +7,67 @@ include::modules/common-attributes.adoc[]
-== Notable changes and improvements in `{odo-title}`
-
-* `{odo-title}` now supports Devfile v2.
-
-* `odo create -s2i` now converts an S2I component into a devfile component. When running `odo create --s2i <component-type>` `{odo-title}` now creates a converted Devfile component based on the S2I images of the specified component type.
+== Notable changes and improvements in `{odo-title}` version 2.5.0
+
+// #5238
+* Creates unique routes for each component, using `adler32` hashing
+// #5252
+* Supports additional fields in the devfile for assigning resources:
+** cpuRequest
+** cpuLimit
+** memoryRequest
+** memoryLimit
+// #5276
+* You can now run the `odo delete` command with the `--deploy` flag to remove components that you deployed using the `odo deploy` command: 
 +
-Note that this feature introduces many breaking changes, see xref:../../cli_reference/developer_cli_odo/odo-release-notes.adoc#odo-known-issues_odo-release-notes[Known Issues] to learn more.
-
-* Operator based service is now created on the cluster only after you run `odo push` and not after `odo service create` anymore.
-
-* You can now use the `--container` flag to specify the container you want to attach storage to when running `odo storage create` command. See xref:../../cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/working-with-storage.adoc#adding-storage-to-a-specific-container_working-with-storage[Adding storage to a specific container] to learn the details.
-
-* `odo catalog component describe` now returns correct JSON if the same name is used for a component in multiple registries.
-
-* Commands that implement changes directly on a cluster now display a message informing a user that `odo push` is not required.
+[source,terminal]
+----
+$ odo delete --deploy
+----
+// #5237
+* Adds mapping support to the `odo link` command
+// #5279
+* Supports ephemeral volumes using the `ephemeral` field in `volume` components
+// #5270
+* Sets the default answer to `yes` when asking for telemetry opt-in
+// #5260
+* Improves metrics by sending additional telemetry data to the devfile registry
+// #5287
+* Updates the bootstrap image to `registry.access.redhat.com/ocp-tools-4/odo-init-container-rhel8:1.1.11`
+// #5308 
+* The upstream repository is available at link:https://github.com/redhat-developer/odo[]
 
-* When creating a component from a devfile, `odo create` now uses a default component name if the name is not specified.
 
-* `odo` now has Telemetry. See xref:../../cli_reference/developer_cli_odo/understanding-odo.adoc#telemetry-in-odo[Telemetry section] to learn how to modify your Telemetry consent preferences.
 
-*  With `odo service`, you can now add or remove custom resource definitions and `ServiceInstance` information in your devfile.   
- 
+[id="odo-fixed-issues_{context}"]
+== Fixed issues
+// #5294
+* Allow `odo deploy` without `env` file
+//5286
+* Fix panic with interactive `odo create` when talking ot a disconnected cluster
```

Comment:
Please add more context to the issues wording them as they are done in the core: https://docs.openshift.com/container-platform/4.9/release_notes/ocp-4-9-release-notes.html#ocp-4-9-bug-fixes

Typo: talking ot a/talking to a